### ISSUE TITLE
Installation instructions are wrong at https://www.swift.org/download/

### ISSUE THREAD

**Issue by dabrahams** [apple/swift-org-website#199]
Quoting from that page:

> Download the [latest package release](https://www.swift.org/download).

Well, presumably these instructions should work for any of the toolchains published, yeah?

> Run the package installer, which will install an Xcode toolchain into /Library/Developer/Toolchains/.
> An Xcode toolchain (.xctoolchain) includes a copy of the compiler, lldb, and other related tools needed to provide a cohesive development experience for working in a specific version of Swift.

It's not clear that it does; at least, having installed a 5.8 development snapshot, this command finds nothing:

```
 find  /Library/Developer/Toolchains/swift-5.8-DEVELOPMENT-SNAPSHOT-2022-12-29-a.xctoolchain -name swiftc
 ```

> Open Xcode’s Preferences, navigate to Components > Toolchains, and select the installed Swift toolchain.

There's no "Preferences" in XCode.  There's "Settings," but no "Components" in there.  You want "XCode > Toolchains"

> Selecting a Swift toolchain affects the Xcode IDE only. To use the Swift toolchain with command-line tools, use xcrun --toolchain swift and xcodebuild -toolchain swift,

nope:
```
$ xcrun --toolchain swift
xcrun: error: no utility name specified
Usage: xcrun [options] <tool name> ... arguments ...
```

also not this:

```
xcrun --toolchain /Library/Developer/Toolchains/swift-5.8-DEVELOPMENT-SNAPSHOT-2022-12-29-a.xctoolchain swift --version
```
(still chooses Swift 5.7)

Dredging my memory for how this is actually done, I open the Info.plist inside the  toolchain package, find a magic toolchain identifier, and try this:

```
$ xcrun --toolchain org.swift.58202212291a swift --version
xcrun: error: can't exec '/Library/Developer/Toolchains/swift-5.8-DEVELOPMENT-SNAPSHOT-2022-12-29-a.xctoolchain/usr/libexec/swift' (errno=Permission denied)
$ TOOLCHAINS=org.swift.58202212291a xcrun swift --version
xcrun: error: can't exec '/Library/Developer/Toolchains/swift-5.8-DEVELOPMENT-SNAPSHOT-2022-12-29-a.xctoolchain/usr/libexec/swift' (errno=Permission denied)
```

>  or add the Swift toolchain to your path as follows:
>
>     $ export PATH=/Library/Developer/Toolchains/swift-latest.xctoolchain/usr/bin:"${PATH}"

Yeah, that does nothing to help me because apparently there's no `swift` executable in there.

**Comment by dabrahams** [comment]
@shahmishal That's a start, but  it didn't _begin_ to address all of the issues with the instructions that I reported.  Are you sure there aren't other things that need to be fixed?

**Comment by shahmishal** [comment]
@dabrahams Good point, @AnthonyLatsis thank you for updating the installation info.
